src/mimefactory.rs

```diff
@@ -544,10 +544,25 @@ impl<'a> MimeFactory<'a> {
             Loaded::Message { .. } => self.msg.rfc724_mid.clone(),
             Loaded::Mdn { .. } => dc_create_outgoing_rfc724_mid(None, &self.from_addr),
         };
-        headers.unprotected.push(Header::new(
-            "Message-ID".into(),
-            render_rfc724_mid(&rfc724_mid),
-        ));
+        let rfc724_mid_headervalue = render_rfc724_mid(&rfc724_mid);
+
+        // Amazon's SMTP server change the `Message-ID`, just as Outlook's SMTP server do.
+        // Outlook adds an `X-Microsoft-Original-Message-ID` header with the original `Message-ID`,
```

Comment:
This is about Outlook the server, not Outlook the MUA?

Reply:
Yes